### PR TITLE
✨Source Aha: Make Connector Compatible with Builder

### PR DESCRIPTION
**Pull request by pabloescoder**
## What
Makes the connector compatible with the Builder UI

## How
- Use Poetry instead of requirements.txt & Dockerfile
- Import schema to Inline Schema in manifest
- Remove $parameters
- Bump minor version
- Make it compatible with the Builder

## User Impact
Non impacting, done for maintainability

## Can this PR be safely reverted and rolled back?
- [x] YES 💚

### CODE REVIEW

**Comment by pabloescoder** [comment]
I've received credentials for this connector, and I tried using those connectors to login to Aha, it shows that the trial period has expired so that's the reason why integration tests for this connector are failing (as the creds are expired). I'll work with Daniel to get them renewed, then we can move ahead with this PR.
image
image

File: docs/integrations/sources/aha.md [hunk 1]
```diff
@@ -38,7 +38,8 @@ Rate Limiting information is updated [here](https://www.aha.io/api#rate-limiting
 ## Changelog
 
 | Version | Date       | Pull Request                                             | Subject                                                                 |
-| :------ | :--------- | :------------------------------------------------------- | :---------------------------------------------------------------------- |
+|:--------|:-----------| :------------------------------------------------------- |:------------------------------------------------------------------------|
+| 0.3.2   | 2024-05-12 | [38144](https://github.com/airbytehq/airbyte/pull/38144) | Make connector compatible with Builder                                  |
```
Comment (girarda) [inline docs/integrations/sources/aha.md:42]:
```suggestion
| 0.3.2   | 2024-05-14 | [38144](https://github.com/airbytehq/airbyte/pull/38144) | Make connector compatible with Builder                                  |
```

**Review by girarda — APPROVED** [review]
approved pending updating the date in the changelog

**Comment by pabloescoder** [comment]
@girarda Updated the date

**Pull request merged**